lib/admin/WP_Auth0_Admin_Generic.php

```diff
@@ -239,13 +243,15 @@ protected function render_field_description( $text ) {
 	 * Output translated dashboard HTML link
 	 *
 	 * @param string $path - dashboard sub-section, if any
+	 * @param bool $append_period - append a period and space to the output
 	 *
 	 * @return string
 	 */
-	protected function get_dashboard_link( $path = '' ) {
-		return sprintf( '<a href="https://manage.auth0.com/#/%s" target="_blank">%s</a>',
+	protected function get_dashboard_link( $path = '', $append_period = FALSE ) {
```

Comment:
I don't think this is necessary or part of this method logic. The method who is calling it must be completing the format. BTW it's only used 2 times on this doc, both of which have a follow up text that can easily contain those ". " you are adding in here. On the contrary, [this line](https://github.com/auth0/wp-auth0/pull/429/files#diff-3a97bffe53fd0d3ebe9560b32470d1d2R217) would have made sense that it called this method passing true since it doesn't have a follow up text, but it's not. 😂

Reply:
The description method auto-adds periods. This is mainly to not screw up translation too much. A translate-able text string that just starts with a period and a space is hard to re-use in any context.

Reply:
Slept on it ... this is goofy. Removing.

Reply:
> A translate-able text string that just starts with a period and a space is hard to re-use in any context.
This is true 100%. But then you can split `". The text to be translated" in `". "` and `"The text to be translated"`, fixing both what I pointed out and the translation inconvenience.

Reply:
But then you're not able to change the period, if needed. Tradeoffs 🤷‍♂️

Reply:
How so?

```
"Text with placeholder on my right" . <placeholder var> . ". " . "Other translatable text".
```

Something like this would work fine. Both the first and last texts could be translated just fine. The dot is a dot, it will never change at least not if you want to keep the original redaction of the message.